src/idl/src/parser.y

```diff
@@ -812,8 +812,11 @@ case_label:
   ;
 
 element_spec:
-    type_spec declarator
-      { TRY(idl_create_case(pstate, LOC(@1.first, @2.last), $1, $2, &$$)); }
+    annotations type_spec declarator
+      { TRY(idl_create_case(pstate, LOC(@1.first, @3.last), $2, $3, &$$));
+        TRY_EXCEPT(idl_annotate(pstate, $$, $1), free($$)); }
+  | type_spec declarator
+      { TRY(idl_create_case(pstate, LOC(@1.first, @2.last), $1, $2, &$$));}
```

Comment:
@optional cannot be applied to union members

Reply:
You are correct, this has been removed.